### ISSUE TITLE
Commenting on code from mobile is broken

### ISSUE THREAD

**Issue by guygastineau** [exercism/exercism.io#3990]
If I try to comment as a mentor or on my own code from my Android device using Brave Browser it doubles the last word I typed when I press space.

**Comment by NobbZ** [comment]
I have similar problems using Chrome.

**Comment by guygastineau** [comment]
Yeah, I think Brave is just modified chrome.

- Do you know of a way around this?

I prefer getting back to students quickly so they can keep their train of thought, but I am not always at my real computer.

**Comment by NobbZ** [comment]
No, I don't see any workaround but dropping the phone and hurrying up to get to a real computer...

**Comment by NobbZ** [comment]
Ps: mobile support is on a long term roadmap

**Comment by iHiD** [comment]
Fixed by https://github.com/exercism/website/pull/285 (not yet deployed)
